tools/block-generator/run_runner.py

```diff
@@ -0,0 +1,201 @@
+import argparse
```

Comment:
I found this pythonized `run_runner.sh` to ease my own testing. In particular it let's you:
* bring up the docker without running the runner
* print out arguments to be run:
  * on the command line
  * for configuring VS-code's debugger thus easing debugging the `block-generator` binary
* purge everything after the fact